srv/_modules/pillar_ops.py

```diff
@@ -63,12 +72,12 @@ def encrypt(decrypt=False):
     return True
 
 
-def _update(data, path, cluster_id, new_passwd, cipher, cipher_key, decrypt):
+def _update(data, path, new_passwd, cipher, cipher_key, decrypt):
```

Comment:
The arguments are confusing to say the least:
Ideally the arguments should convey what data-type and value they hold
Instead of
`data, path, new_passwd, cipher, cipher_key, decrypt` 
we should have
`pillar_info, pillar_name, secret_passwd, cipher_obj, cipher_key, is_decrypt`
This can be later improvement, which we can handle later as well....

Reply:
Okay